plugins/module_utils/kibana.py

```diff
@@ -73,61 +73,80 @@ def get_alert_connector_type_by_name(self, connector_type_name):
     connector_types = self.send_api_request(endpoint, 'GET')
     return next(filter(lambda x: x['name'] == connector_type_name, connector_types), None)
   
-  # Elastic Rules functions
+  # Elastic Security Rules functions
 
-  def update_rule(self, body):
+  def update_security_rule(self, body):
     endpoint = "detection_engine/rules"
-    update_rule = self.send_api_request(endpoint,'PATCH',data=body)
+    update_rule = self.send_api_request(endpoint, 'PATCH', data=body)
     return update_rule
 
-  def get_rules(self,page_size,page_no):
+  def get_security_rules(self, page_size, page_no):
     endpoint = "detection_engine/rules/_find?page=" + str(page_no) + "&per_page=" + str(page_size)
     rules = self.send_api_request(endpoint, 'GET')
     return rules
+
+  def get_security_rules_byfilter(self, rule_name):
+    page_no = 1
+    page_size = 100
+    filter_scrubbed = str(rule_name).replace(" ", "%20")
```

Comment:
Should probably use urllib.parse.quote() here rather than doing a string replacement like that, will deal with other control characters if they happen to be included

https://docs.python.org/3/library/urllib.parse.html